pkg/trace/api/telemetry.go

```diff
@@ -60,8 +60,17 @@ func (r *HTTPReceiver) telemetryProxyHandler() http.Handler {
 		return http.NotFoundHandler()
 	}
 
+	underlyingTransport := r.conf.NewHTTPTransport()
+	// Fix and documentation taken from pkg/trace/api/profiles.go:121
```

Comment:
Listing the line number in this way is a bit brittle, since changes to this file will change this line number over time.

```suggestion
	// Based on the profiler's proxy in pkg/trace/api/profiles.go
```